packages/utilities/gel-foundations/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@bbc/gel-foundations",
-  "version": "0.3.0",
+  "version": "0.3.1",
```

Personally I think this is a much bigger change than a `patch` so would suggest `0.4.0` or maybe even `1.0.0`. Would be keen to hear what others think as well

---

I agree, I would go for 
```suggestion
  "version": "1.0.0",
```